Jenkinsfile

```diff
@@ -117,7 +118,7 @@ pipeline {
               RELEASE_VERSION="$(npm run --silent current-version --workspace=iframe-coordinator)"
               npm install --no-progress -P -E iframe-coordinator@$RELEASE_VERSION --workspace=iframe-coordinator-cli
               git add . && git commit --amend --no-edit --no-verify
-              git tag -a v$RELEASE_VERSION -m "chore(release): $RELEASE_VERSION"
+              git tag -fa v$RELEASE_VERSION -m "chore(release): $RELEASE_VERSION"
```

Comment:
This is probably why the versions got out of sync. Thi command was failing as the tag was created in the `sh "npm run release"` step.